selfdrive/controls/controlsd.py

```diff
@@ -579,7 +582,7 @@ def state_control(self, CS):
     # Check which actuators can be enabled
     CC.latActive = self.active and not CS.steerFaultTemporary and not CS.steerFaultPermanent and \
                    CS.vEgo > self.CP.minSteerSpeed and not CS.standstill
-    CC.longActive = self.active and not self.events.any(ET.OVERRIDE_LONGITUDINAL) and self.CP.openpilotLongitudinalControl
+    CC.longActive = self.enabled and not self.events.any(ET.OVERRIDE_LONGITUDINAL) and self.CP.openpilotLongitudinalControl
```

Comment:
changed to enabled (which includes preEnabled) so that we can apply brake while pre-enabling at a standstill to keep the vehicle stopped